src/main.cc

```diff
@@ -33,6 +33,59 @@ DEFINE_bool(license, false, "Output program license");
 DEFINE_bool(config, false, "Show config");
 DEFINE_bool(path, false, "Show current path");
 
+void printIcon(bool large) {
+    if (large) {
+        std::string icon = "\n"
+                           "    +-------------------+  │\n"
+                           "+-------------------+###|  │  Lightening Slide V0.1a\n"
```

Comment:
It isn't a good idea to write version text directly inside the program, I think.
Well, there is something like `GFLAGS_NAMESPACE::SetVersionText`

Reply:
What's that

Reply:
Maybe you should explain how `GFLAGS_NAMESPACE::SetVersionText` so that I can fix this.

Reply:
solved